src/test/regress/expected/multi_json_agg.out

```diff
@@ -0,0 +1,183 @@
+--
+-- MULTI_JSON_AGG
+--
+SET citus.next_shard_id TO 520000;
```

Comment:
Setting the GUC is not really necessary unless you're creating new distributed tables.

Reply:
I can clean that up

Reply:
No worries, I'll merge this PR and then fix it up.